README.md

```diff
@@ -36,7 +36,7 @@ Once a pull request is submitted, multiple committers may comment on it and prov
 
 ### Reviewing Pull Requests
 
-Once a pull request has been submitted and the correct label assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committers. You can read more about the review process and expectations [here](https://developers.mattermost.com/contribute/getting-started/code-review/). 
+Once a pull request has been submitted and the correct label assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committees. You can read more about the review process and expectations [here](https://developers.mattermost.com/contribute/getting-started/code-review/). 
```

Comment:
```suggestion
Once a pull request has been submitted and the correct label assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committers. You can read more about the review process and expectations [here](https://developers.mattermost.com/contribute/getting-started/code-review/). 
```

The maintainers on the Mattermost project are known as "core committers". You can see the list [here](https://developers.mattermost.com/contribute/getting-started/core-committers/). :)